build.xml

```diff
@@ -148,7 +148,9 @@
 
   <target name="run" depends="jar"
           description="Runs the development version of the program">
-    <java jar="${jar.dir}/${productname}.jar" fork="true"/>
+          <echo message="To run the demonstration:"/>
+          <echo message="Build using 'ant jar'"/>
+          <echo message="Run using 'java -jar build\jar\Redistricting.jar'"/>
```

Comment:
Please construct this comment dynamically based on the configured directory and product name. (You can use the previous java task definition as a template.)